docs/user/library/data/mongodb.rst

```diff
@@ -51,8 +51,9 @@ following connection parameters:
 * ``schema_store``: Designates the storage for inferred and manually defined 
   schemas.
 
-  This field can accept either a MongoDB or file URI. The directory will be 
-  created if it does not exist, in which case write permissions will be necessary.
+  This field can accept either a MongoDB or file URI or Http Url to JSON file or Zip file
```

Comment:
The URL can be either HTTP or HTTPS, I would also explain better what the content of the ZIP shoudl look like and mentioning that in this case the schema store will work as read only. If a collection that is not available tries to be used then it will fail with an exception (at least this sit he behavior I expect).

Reply:
understood

the current behavior is that if collection is not found, its ignored producing a warning log message